src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStr) {
+    let count;
+
+    do {
+      count = 0;
```

Comment:
we can replace `count` variable with some boolean variable for better readability

what do you think?

Reply:
it is possible. I changed